apps/datahub/src/app/app.module.ts

```diff
@@ -176,6 +176,9 @@ export const metaReducers: MetaReducer[] = !environment.production ? [] : []
           return fetch(getOptionalSearchConfig().FILTER_GEOMETRY_URL)
             .then((resp) => resp.json())
             .then(getGeometryFromGeoJSON)
+            .catch(() =>
+              console.warn('The geometry filter failed to load its geometry')
```

Comment:
I think we can even make this a console log saying "No spatial filter was applied since a valid geometry could not be found". This is going to be the nominal situation for anonymous users on geOrchestra, so outputting a warning might make it seem like there's a problem to be fixed.